helm/pv-migrate/templates/rsync/job.yaml

```diff
@@ -35,7 +35,7 @@ spec:
             - |
               n=0
               rc=1
-              retries=10
```

Comment:
Can we rename this to `maxRetries`? The word `attempt` is used with a different meaning in the project.

Also, would you mind also making the sleep seconds configurable as well to the following lines?
```
                echo "rsync attempt $n/10 failed, waiting 5 seconds before trying again"
                sleep 5
```

Maybe a new value like `retryPeriodSeconds` (similar to the Kubernetes probes' naming)

And one final thing: `$n/10` needs to be `$n/$retries`